src/index.js

```diff
@@ -76,31 +77,41 @@ $(document).ready(function() {
         // projectName variable is defined in our example projects so the
         // correct test suite is automatically loaded. This sets default text
         // for <option> text and project indicator in top right corner.
+
         if (typeof projectName !== 'undefined') {
           projectNameLocal = projectName;
         }
 
+        let toggleElement = document.getElementById('toggle');
+        let indicatorWrapper = document.getElementById(
+          'fcc_test_suite_indicator_wrapper'
+        );
+        let placeholder = document.getElementById('placeholder');
+
         if ((!projectNameLocal) && (projectTitleCase === null)) {
-          document.getElementById('placeholder').innerHTML = '- - -';
-          document.getElementById(
-            'fcc_test_suite_indicator_wrapper'
-          ).innerHTML = '';
+          placeholder.innerHTML = '- - -';
+          indicatorWrapper.innerHTML = '';
         } else if (projectNameLocal) {
-          document.getElementById('placeholder').innerHTML =
+          placeholder.innerHTML =
             `${localStorage.getItem('example_project')}`;
-          document.getElementById(
-            'fcc_test_suite_indicator_wrapper'
-          ).innerHTML =
+          indicatorWrapper.innerHTML =
             '<span id=fcc_test_suite_indicator>FCC Test Suite: ' +
             `${localStorage.getItem('example_project')}</span>`;
         } else {
-          document.getElementById('placeholder').innerHTML = projectTitleCase;
-          document.getElementById(
-            'fcc_test_suite_indicator_wrapper'
-          ).innerHTML =
+          placeholder.innerHTML = projectTitleCase;
+          indicatorWrapper.innerHTML =
             '<span id=fcc_test_suite_indicator>FCC Test Suite: ' +
             `${projectTitleCase}</span>`;
         }
+        let visibleProjectName = placeholder.innerHTML.split(' ').join('');
+        // If this is the first time loading this project, show test window
+        if (!localStorage.getItem('fCC_' + visibleProjectName + '_hide')) {
```

Comment:
Searching on `'fCC_' + visibleProjectName + '_hide'` is a bit awkward. Could you assign this string to a variable and use the same variable name everywhere? I think this could even deserve a function that builds the name so that logic is in one place. DRY principle.